src/feature/greeting/greetingSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  isLoading: false,
+  greetings: {},
+  error: undefined,
+};
+
+export const fetchGreeting = createAsyncThunk('greeting/fetchGreeting', async () => {
+  const response = await axios.get('http://127.0.0.1:3000/api/v1/greeting');
+  return response.data;
+});
+
+const greetingSlice = createSlice({
+  name: 'greeting',
+  initialState,
+  extraReducers: (builder) => {
+    builder.addCase(fetchGreeting.pending, (state) => {
+      state.isLoading = true;
+    })
+      .addCase(fetchGreeting.fulfilled, (state, action) => {
+        state.isLoading = false;
+        state.greetings = action.payload;
+      })
```

Comment:
Hello @zbahati I am so sorry but I didn't notice it I searched for it in the feature directory so beside the slice but not find it but sorry for this inconvenience! please submit it again and it will be approved 🔥